deps.toml

```diff
@@ -27,7 +27,7 @@ apache-commons = { module = "org.apache.commons:commons-compress", version = "1.
 apache-commons-lang = { module = "org.apache.commons:commons-lang3", version = "3.11" }
 appender-log4j2 = { module = "com.therealvan:appender-log4j2", version = "3.6.0" }
 assertj-core = { module = "org.assertj:assertj-core", version = "3.21.0" }
-aws-java-sdk-s3 = { module = "com.amazonaws:aws-java-sdk-s3", version = "1.12.6" }
+aws-java-sdk-s3 = { module = "com.amazonaws:aws-java-sdk-s3", version = "1.12.746" }
```

Comment:
This deps.toml entry isn't actually used anywhere, instead, you need to bump the versions in various build.gradle files: 
- `airbyte-cdk/java/airbyte-cdk/s3-destinations/build.gradle`
- `airbyte-integrations/connectors/destination-redshift/build.gradle`
- `airbyte-integrations/connectors/destination-s3/build.gradle`

Reply:
@postamar Thank you for reviewing. I made the additional changes in the specified places. Please take a look

Reply:
you'll also need to bump the version for each of the connector metadata.yaml file and update the associated readme files in the docs/integrations directory. We have some ideas on how to streamline that annoying part of modifying the code, but haven't found the time to work on it yet

Reply:
@stephane-airbyte 
Can you describe in more detail what needs to be done with regard to the readme and meatadata.yml files?
Would I really need to update every connector? 
I'm not really an airbyte user, so I'm not completely sure how the entire stack works together. What I want to do is enable the airbyte worker/server/orchestrator pods to use  EKS pod identity through a service account. Do all the connectors need to be updated for this to happen?

Reply:
only the 3 connectors you're modifying would need to have their metadata.yaml and their READMEs updated.
In metadata.yaml (in the connector directory), you simply need to update the value of dockerImageTag by bumping the version (a patch version bump seems appropriate for a library upgrade)
In `docs/connectors/{source/destination}/connector.md`, you simply need to update the changelog to add a new entry to describe your changes.
Once that's done and your changes are merged, our automation will create a new docker image for all the upgraded connectors, which your airbyte deployment should pick up automatically.

Reply:
@stephane-airbyte Thanks for giving some more explicit details.
While I am happy to do this, I just wanted to double check something we are on the same page about something:
Implementing the newer version of the AWS java sdk is not meant to enable EKS pod identity for airbyte connectors, rather to enable airbyte external logging as described [here](https://docs.airbyte.com/deploying-airbyte/on-kubernetes-via-helm#external-logs-with-s3). Using EKS pod identity here would allow me to use IAM roles directly instead of AM credentials. 
Are we aligned in that? I realized I failed to call out that specific use case in the PR description.